crates/matrix-sdk-crypto/src/file_encryption/attachments.rs

```diff
@@ -238,10 +238,12 @@ impl<'a, R: Read + ?Sized + 'a> AttachmentEncryptor<'a, R> {
             kty: "oct".to_owned(),
             key_ops: vec!["encrypt".to_owned(), "decrypt".to_owned()],
             alg: "A256CTR".to_owned(),
+            #[allow(clippy::unnecessary_to_owned)]
             k: Base64::new(key.to_vec()),
             ext: true,
         });
-        let encoded_iv = Base64::new((iv).to_vec());
+        #[allow(clippy::unnecessary_to_owned)]
+        let encoded_iv = Base64::new(iv.to_vec());
```

Comment:
Is clippy emitting false positives here?

Reply:
It is.

Reply:
Alright. Hadn't gotten false positives from that particular lint myself. Looks like https://github.com/rust-lang/rust-clippy/issues/9351 is tracking this.

Reply:
Yeah it just started happening today. So it's pretty new.